### ISSUE TITLE
Unofficial Third-Party Xbox One S N-1 Controller Does Not Work

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#413]
As brought up from this discussion: https://github.com/LukeShortCloud/winesapOS/discussions/405

**Comment by LukeShortCloud** [comment]
@jpbrisa Good news, I believe I am close to solving how to get your Xbox One S controller to work!

```
```

**Comment by LukeShortCloud** [comment]
You reported that this is the device ID that Linux sees:

```
Bus 001 Device 006: ID 045e:02ea Microsoft Corp. Xbox One S Controller
```

This is fully supported via the `xpad` driver and it even has a workaround to deal with some strange behavior of that specific controller.

https://github.com/paroj/xpad/blob/0270e3759fccdd2887553fa4fd2cd07844b29f49/xpad.c#L161
https://github.com/paroj/xpad/blob/0270e3759fccdd2887553fa4fd2cd07844b29f49/xpad.c#L588
https://linux-hardware.org/?id=usb:045e-02ea

The Linux kernel ships its own older/outdated `xpad` driver and winesapOS provides a newer `xone` driver. Here are possible problems and solutions:
1. [xpad-noone](https://aur.archlinux.org/packages/xpad-noone-dkms) needs to be installed. This will override the Linux kernel's `xpad` with one that removes support for Xbox One controllers. Instead, it will only handle original Xbox and Xbox 360 controllers. Then the `xone` driver is used for Xbox One controllers.
2. The newer [xpad](https://github.com/paroj/xpad) driver needs to be installed.
    1. This may cause conflicts (and may already be causing conflicts) with the `xone` driver.
    2. We could remove the `xone` driver but it is overall better suited for newer Xbox One devices than `xpad` is.

If you need help trying these solutions, let me know and I can provide some more guidance.

**Comment by LukeShortCloud** [comment]
@jpbrisa This has now been fixed. We went with solution number 1. This fix will be included in our upcoming stable 3.2.0 release later this month. Thank you again for reporting your controller issue so that we can resolve it!
